### ISSUE TITLE
Add more project basic metrics

### ISSUE THREAD

**Issue by VelizarVESSELINOV** [isaacs/github#825]
- Lines of code
- Code duplication

We should avoid to use extra plugins to have basic metric like SLOC: https://addons.mozilla.org/en-US/firefox/addon/github-sloc/

**Comment by nealmcb** [comment]
Yes - NCSL / SLOC would be an excellent addition to the Insights section of the site.
Insights should also list the primary languages that the code is written in.
There is another project out there that does that for lots of open source projects, but I can't remember its name now....
